tests.py

```diff
@@ -110,6 +110,11 @@ def isSocketClosed(self, a):
             msg = a.recv(1)
         except socket.timeout:
             is_closed = False
+        except socket.error as e:
```

Comment:
Looks like it works for other people without this change on Linux. Are you sure it is necessary?